unit_tests/src/test_atomic_apply_acl_operations.cpp

```diff
@@ -174,6 +175,143 @@ TEST_CASE("atomic_apply_acl_operations")
     }
 }
 
+TEST_CASE("Administrators are allowed to use the admin_mode option")
+{
+    //
+    // This test requires that the environment represent a rodsadmin user.
+    //
+
+    using namespace std::string_literals;
+
+    irods::experimental::client_connection conn;
+
+    // Create a test rodsuser.
+    ua::user test_user{"unit_test_test_user"};
+    irods::at_scope_exit remove_user{[&conn, &test_user] { ua::client::remove_user(conn, test_user); }};
+    CHECK(ua::client::add_user(conn, test_user).value() == 0);
```

Comment:
Should this be `REQUIRE` since it's setting up the real test?

Reply:
It can be. Will change.

Reply:
Done.